public/javascripts/app/navi.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Created by AGromov on 20.07.2016.
+ */
+define([
+    "jquery",
+    "handlebars",
+    "text!view/menu.hbs"
+], function($, hbs, tpl) {
+    return {
+        init: function() {
+            this.render([{title: "corrona_cdm"}]);
```

Comment:
Please remove the name that looks like any of our customers. Use something general.